### ISSUE TITLE
Pull Warning: stripos() expects parameter 1 to be string

### ISSUE THREAD

**Issue by nypolitics** [10up/distributor#369]
I'm trying to pull posts from a minor site to the main site. When I attempt to pull, I get this warning: "Warning: stripos() expects parameter 1 to be string, array given in /wp-includes/formatting.php on line 4185"

**Comment by jeffpaul** [comment]
@nypolitics are you using Distributor in a multisite setup?  If not, are the External Connections setup and showing a green circle as connected?

**Comment by nypolitics** [comment]
I am not using a multisite setup (that's another story - problems accessing sub sites). Both the main and external site are connected with a green circle. When I push content from the sub site to the main site, the post successfully transfers. However, when I try to push content from the main site to the sub site, Distributor on the main site says the post was successfully transferred, but the post does not appear on the sub site.







---- On Thu, 18 Apr 2019 16:43:07 -0400 Jeffrey Paul <notifications@github.com> wrote ----



https://github.com/nypolitics are you using Distributor in a multisite setup?  If not, are the External Connections setup and showing a green circle as connected?

**Comment by jeffpaul** [comment]
@nypolitics are you trying to push a Post, Page, or Custom Post Type from the main site to the sub site and not seeing it appear on the sub site?  I'm trying to understand specifically what you're trying to push that's not working.  In parallel, can you try on the sub site to Pull that same piece of content from the main site and see if that works?

**Comment by jeffpaul** [comment]
In email discussions with @nypolitics it appears that a potential root cause here could be that one External Connection is on HTTPS while the other is on HTTP.  We'll need to test this to see if that causes issues with pushing or pulling posts.  Additional details via email:

> I'm now able to pull posts from the main site (on HTTPS) to the remote site (on HTTP). Although the posts are pulled, they cannot be viewed. Each are listed with a Warning:
"Warning: stripos() expects parameter 1 to be string, array given in /server/path/remote-site/wp-includes/formatting.php on line 4185."
> 
> When I try to view the post, I get this error in my browser window: "array’s server IP address could not be found." The location bar content: /array
